chapter_5_practic/task6.cpp

```diff
@@ -0,0 +1,59 @@
+// Start with the program from Exercise 11 in Chapter 4, �Structures, � which adds two
+// struct time values.Keep the same functionality, but modify the program so that it uses
+// two functions.The first, time_to_secs(), takes as its only argument a structure of type
+// time, and returns the equivalent in seconds(type long).The second function,
+// secs_to_time(), takes as its only argument a time in seconds(type long), and returns a
+// structure of type time.
+
+#include <iostream>
+
+struct user_time
+{
+	int hours = 0;
+	int minutes = 0;
+	int seconds = 0;
+};
+
+user_time time1;
+user_time time2;
+
+const int sec_in_hour = 3600;
+const int sec_in_min = 60;
+char ch = ' ';
+long totalsec = 0;
+
+long time_to_secs(user_time);
```

Comment:
It's better to name parameters in function signature as well

Reply:
Fixed
https://github.com/Dmitriy-Romanchuk/master/pull/5/commits/799c73b05317bbba47a2b293b1aab739bbd44172